analyse_data.py

```diff
@@ -79,6 +80,11 @@ def print_common_failure_reasons(build_data):
     print(group_by_classification(get_classified_failures(build_data)))
 
 
+def print_common_failure_daily(build_data):
+    print("Daily drill-down on failure classifications:")
+    print(get_daily_classification_pivot(get_classified_failures(build_data)))
```

Comment:
:question: This recalculates the classification from the previous section.
Saving the intermediate state might be a nice optimisation.